### PR TITLE
Add Filen

### PR DESCRIPTION
**Pull request by ghost**
Hey, sorry if this feels a bit spammy, but this is my first time submitting a pull request to a public repo 😅

Anyways, I wanted to add an entry for Filen.io because it doesn't get picked up by 1Password

### CODE REVIEW

**Comment by person1234-ux** [comment]
@AlexanderCormi4 look good your images just need to be changes to a svg file look at the pull request I created on your branch

**Comment by AlexanderCormi4** [comment]
Once I merge your pull request, what do I do next? Do I create another pull request here with the updated images?

**Comment by person1234-ux** [comment]
the changes are automatic

**Comment by AlexanderCormi4** [comment]
@person1234-ux, I just merged your pull request. What do I do next?

**Comment by ghost** [comment]
Oh okay, thanks!

**Comment by phallobst** [comment]
I don't think that new commits are automatically added to PRs if you are using the `master` branch in your own repository (which you did here).

**Comment by ghost** [comment]
Ah, my bad. Do I revert the pull request and merge it here?

**Comment by phallobst** [comment]
I was wrong about the PR update, so I deleted my comment. :wink:

**Comment by ghost** [comment]
There's nothing left to do, right?

**Comment by person1234-ux** [comment]
Yah there is nothing left to do

**Comment by ghost** [comment]
Awesome, learned a lot here 🙌

File: img/f/filen.io.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...7.7-10.4 37.1 7.9 77.7 42.5 94.3 12.2 5.9 21.3 7.9 35.5 7.9 14.7 0 23.7-2.1 36.5-8.5 20.3-10.1 36.1-29.5 42.1-51.9 2.8-10.2 2.6-31.2-.4-41.4-4-14-10.4-24.9-20.7-35.2-9.4-9.5-16.4-14.2-27.9-18.8-13.4-5.4-31.4-7-45.6-4.1zm30.2 11.6c2.9.6 8.6 2.5 12.6 4.1l7.2 3-.2 15.7-.3 15.6-37.7.3L67 67v10.3c0 8.3-.4 11-2 14.2-2.2 4.3-1.9 6.8 1.3 10.3 5.7 6.2 15.7-.7 12.8-8.9-.6-1.8-1.1-6.1-1.1-9.5V77h75V59.7l.1-17.2 6.7 7c10.3 10.8 16.6 24.6 17.9 39.2l.6 7.3H143v60.1l-6.2 2.9-6.3 3-.3-25.6c-.2-24.3-.1-25.7 1.8-28.1 4.3-5.5 1.4-14.4-5.1-15.8-9-2-15.3 8.3-9.4 15.3l2.5 3v53l-2.2.6c-7 1.8-25.7-.5-34.3-4.3l-4.5-2V120h6.9c4 0 7.3.5 7.7 1.1.3.6 2.2 1.7 4 2.5 2.9 1.2 3.9 1.2 6.8 0 4.1-1.7 5.6-4.2 5.6-9.1 0-7.4-9.1-11.3-15.3-6.5-2.2 1.7-4 2-14.1 2H69l-.2 21.1-.3 21.2-5.4-4.9c-3-2.7-6.7-6.7-8.3-8.8l-2.8-3.9v-78l2.6-3.4 2.5-3.3H84c18.5 0 27.1.3 27.6 1.1.3.6 2.2 1.7 4 2.5 2.9 1.2 3.9 1.2 6.8 0 4-1.7 5.6-4.2 5.6-8.8 0-7.6-10-11.7-15.4-6.3-1.2 1.2-5.1 1.5-22.8 1.5-19.6-.1-21.1-.2-19.3-1.7 3.4-2.7 16.1-8.3 22.2-9.7 7.1-1.7 23.3-2 30.5-.5zM42.5 96c0 16.7-1.3 20.2-2.2 5.7-.6-8.7.4-21.7 1.5-21.7.4 0 .7 7.2.7 16zM177 108.2c0 4.5-5.8 18.7-10.3 25.4-2.5 3.8-6.6 8.9-9.1 11.4l-4.6 4.4V106h12c11.8 0 12 0 12 2.2z"/></svg>
```
Comment (phallobst) [inline img/f/filen.io.svg:1]: Here's a cleaner version without padding:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="-3 -3 685.4 685.4"><circle cx="190.3" cy="332" r="34.3"/><circle cx="375" cy="123" r="31"/><circle cx="398.5" cy="356.7" r="40.4"/><circle cx="301" cy="413.4" r="33.3"/><g fill="none" stroke="#000" stroke-width="42"><path d="M190 619.5v-206h85.5M75 520.3V157"/><path stroke-linecap="square" d="M115 123.7h225"/><path d="M494 62.4v170H190v73m208.2 340V385M500 608V356h155"/><circle cx="339.5" cy="339.7" r="321.7" stroke-linecap="square" stroke-linejoin="round"/></g></svg>
```

**Review by phallobst — CHANGES_REQUESTED** [review]
The documentation page isn't really that helpful, but better than nothing, I guess. :man_shrugging:

**Comment by Carlgo11** [comment]
As the requested changes haven't been resolved yet I'm going to label this PR as inactive.
Unless there's activity within the next week I'll close the PR.

**Comment by person1234-ux** [comment]
@AlexanderCormi4 please allow the changes that they have requested to go through

**Comment by ghost** [comment]
Do I just press resolve?

**Comment by noahggarrett-1** [comment]
> Do I just press resolve?

I believe you have to commit the changes and then you can press resolved
@AlexanderCormi4

**Comment by noahggarrett-1** [comment]
click on the red think next to phallobst name then click approve changes. Sorry it has been a minutes sense I have done that

**Comment by ghost** [comment]
@noahggarrett-1 Thanks

**Comment by phallobst** [comment]
If you want to use the logo that I suggested, you need to use "Commit suggestion" - clicking "Resolve conversation" will close it with no further action. I have re-opened it for your convenience.

**Comment by Carlgo11** [comment]
Closing due to inactivity.

**Pull request closed without merging**